probe-rs-target/src/memory.rs

```diff
@@ -127,6 +131,15 @@ impl MemoryRange for Range<u64> {
                 || range.contains_range(self)
         }
     }
+
+    fn align_to_32_bits(&mut self) {
+        if self.start % 4 != 0 {
+            self.start -= self.start % 4;
+        }
+        if self.end % 4 != 0 {
+            self.end += 4 - self.end % 4;
```

Comment:
What happens if `end = u32::MAX`? I think that will overflow.

Reply:
Good catch ... I will fix :)